bbc/simorgh
# Pull in new Inline link styling

Resolves #1710

**Overall change:** Branch for testing the new UX link styling being added as part of https://github.com/bbc/psammead/pull/707. This PR adds the test fixture called `ctestreith0o` EG: http://localhost:7080/news/articles/ctestreith0o

**Code changes:**

- Updates the UX styling to the desired designs as per https://github.com/bbc/psammead/pull/707

Testing this:
- Ensure the special characters in mentioned in this [issue](https://github.com/bbc/simorgh/issues/1724) are fixed in the example story https://www.test.bbc.co.uk/news/articles/cev804m5n0vo (might need to copy this into local fixture data, can use the JSON endpoint to get the payload https://www.test.bbc.co.uk/news/articles/cev804m5n0vo.json)
- Visual regression of font, especially on components such as inline link.
- Check firefox high contrast mode (Go to FireFox -> Preference -> Language & Appearance -> Colours -> 'Override the colours ...' set as `Always`)
- Check Windows high contrast mode
- Check with page zoomed to high percentages (300%+)
- Check across different breakpoints 
- Check across different browsers

---
